packages/client/src/rpc/validation.ts

```diff
@@ -532,6 +532,54 @@ export const validators = {
     }
   },
 
+  get consolidationRequest() {
+    return (requiredFields: string[] = ['sourceAddress', 'sourcePubkey', 'targetPubkey']) => {
+      return (params: any[], index: number) => {
+        if (typeof params[index] !== 'object') {
+          return {
+            code: INVALID_PARAMS,
+            message: `invalid argument ${index}: argument must be an object`,
+          }
+        }
+
+        const wt = params[index]
```

Comment:
What does `wt` indicate here?

Reply:
just a variable reference to the request